test/functional/causal_consistency_tests.js

```diff
@@ -25,7 +24,11 @@ describe('Causal Consistency', function() {
     test.client = this.configuration.newClient({ w: 1 }, { poolSize: 1, auto_reconnect: false });
     return test.client.connect();
   });
-  afterEach(() => test.client.close());
+
+  afterEach(() => {
+    test.listener.uninstrument();
```

Comment:
unrelated: should we be testing with `uninstrument`?

Reply:
probably not, but seems orthogonal to the goal of this PR